app/controllers/verify/usps_controller.rb

```diff
@@ -2,21 +2,42 @@ module Verify
   class UspsController < ApplicationController
     include IdvStepConcern
 
-    before_action :confirm_step_needed
+    before_action :confirm_mail_not_spammed
 
     def index
       @applicant = idv_session.normalized_applicant_params
+      @title = title
+      @button = button
     end
 
     def create
+      unless mail_spammed?
+        flash[:error] = t('idv.errors.mail_limit_reached') unless mail_not_spammed?
+        redirect_to verify_review_url
```

Comment:
reek is getting made about calling `redirect_to verify_review_url` twice, but this could actually be a double render error. This needs to be `return redirect_to verify_review_url`